server/src/main/java/org/opensearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -400,9 +400,9 @@ public int slop() {
     /**
      * Sets the fuzziness used when evaluated to a fuzzy query type. Defaults to "AUTO".
      */
-    public MultiMatchQueryBuilder fuzziness(Object fuzziness) {
```

Comment:
If we want to backport that to 2.x, we may need to keep `public MultiMatchQueryBuilder fuzziness(Object fuzziness)` but deprecate it (otherwise it is breaking change).